src/agent0/ethpy/base/transactions.py

```diff
@@ -499,7 +546,7 @@ async def _async_send_transaction_and_wait_for_receipt(
     unsent_txn["nonce"] = nonce
 
     signed_txn = signer.sign_transaction(unsent_txn)  # type: ignore
-    tx_hash = web3.eth.send_raw_transaction(signed_txn.raw_transaction)
+    tx_hash = web3.eth.send_raw_transaction(signed_txn.rawTransaction)
```

Comment:
This seems weird, `eth_account` made an update to change this variable name. Looks like it's resolving to v0.10.1, which isn't the newest version

Reply:
can you pin the `eth_account` version to the latest?

Reply:
Resolved, see comments in `pyproject.toml`